docs/FAQ/how-deploy-bold-bi-in-eks-using-network-load-balancer-with-ssl-certificate-from-acm.md

```diff
@@ -0,0 +1,245 @@
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Create Namespace
+
+Run the following command to create the namespace where the Bold BI resources will be deployed in the Kubernetes cluster. The default namespace is <i>bold-services</i>.
+
+```console
+kubectl create ns bold-services
+```
+
+## Install Chart
+
+For the Helm chart, you need to craft a `values.yaml` file. So download the values.yaml file from [here](https://raw.githubusercontent.com/boldbi/boldbi-kubernetes/UMP-22952-Autodeployment-documentation/helm/custom-values/eks-values.yaml) and make needed changes based on your cluster provider.
+
+The following table allows you to craft the values.yaml file with required values for Bold BI deployment. so please read the description carefully and enter the values in values.yaml file.
+
+<br/>
+
+<table>
+    <tr>
+      <td>
+       <b>Name</b>
+      </td>
+      <td>
+       <b>Description</b>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       namespace
+      </td>
+      <td>
+       The namespace in which the Bold BI resources will be dpleoyed in the kubernetes cluster.<br/>
+       The default namespace is <i>bold-services</i>
+      </td>
+    </tr>
+    <tr>
+      <td>
+       appBaseUrl *
+      </td>
+      <td>
+       Domain with https protocol.
+       <br/>
+       Ex: `https://example.com`
+      </td>
+    </tr>
+    <tr>
+      <td>
+       optionalLibs
+      </td>
+      <td>
+       These are the client libraries used in Bold BI by default.<br/>
+       '<i>mongodb,mysql,influxdb,snowflake,oracle,google,clickhouse</i>'<br/>
+       Please refer to <a href='docs/configuration.md#client-libraries'>Optional Client Libraries</a> section to know more.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       clusterProvider
+      </td>
+      <td>
+       The type of kubernetes cluster provider you are using.<br/>
+       The supported values are '<i>gke,eks and aks</i>'
+       Please refer to <a href='configuration.md#cluster-provider'>Cluster Provider</a> section to know more.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       persistentVolume*
+      </td>
+      <td>
+       This is a file storage information to store the shared folders for application usage.
+       Please refer to <a href='configuration.md#persistent-volume'>this</a> section to know more on how to set Persistant Volumes for Bold BI.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       loadBalancer
+      </td>
+      <td>
+       Currently we have provided support for Nginx and Istio as Load Balancers in Bold BI. Please refer to <a href='docs/configuration.md#load-balancing'>this</a> section for configuring Load balancer for Bold BI.
+      </td>
+    </tr>
+    <tr>
+      <td>
+       autoscaling
+      </td>
+      <td>
+       By default autoscaling is enabled in Bold BI. Please refer to <a href='configuration.md#auto-scaling'>this</a> section to configure autoscaling in Bold BI.
```

Comment:
By default, autoscaling is enabled in Bold BI. Please refer to <a href='configuration.md#auto-scaling'>this</a> section to configure autoscaling in Bold BI.

Reply:
concern addressed